website/docs/r/data_factory_dataset_cosmosdb_mongoapi.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+subcategory: "Data Factory"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_factory_dataset_cosmosdb_mongoapi"
+description: |-
+  Manages an Azure Cosmos DB Mongo API Dataset inside an Azure Data Factory.
+---
+
+# azurerm_data_factory_dataset_cosmosdb_mongoapi
+
+Manages an Azure Cosmos DB Mongo API Dataset inside an Azure Data Factory.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_data_factory" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_data_factory_linked_service_cosmosdb_mongoapi" "example" {
+  name              = "example"
+  data_factory_id   = azurerm_data_factory.example.id
+  connection_string = "mongodb://accname:secretpass@foobar.documents.azure.com:10255"
+  database          = "foo"
+}
+
+resource "azurerm_data_factory_dataset_cosmosdb_mongoapi" "example" {
+  name                = "example"
+  collection_name     = "bar"
+  data_factory_id     = azurerm_data_factory.example.id
+  linked_service_name = azurerm_data_factory_linked_service_cosmosdb_mongoapi.example.name
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `collection_name` - (Required) The collection name of the Data Factory Dataset Azure Cosmos DB Mongo API.
+
+* `data_factory_id` - (Required) The Data Factory ID in which to associate the Linked Service with. Changing this forces a new resource.
+
+* `linked_service_name` - (Required) The Data Factory Linked Service name in which to associate the Dataset with.
+
+* `name` - (Required) Specifies the name of the Data Factory Dataset. Changing this forces a new resource to be created. Must be globally unique. See the [Microsoft documentation](https://docs.microsoft.com/azure/data-factory/naming-rules) for all restrictions.
+
+---
+
+* `additional_properties` - (Optional) A map of additional properties to associate with the Data Factory Dataset.
```

Comment:
As per `parameters`, can we make this a little more clear as to what is expected here, and what is supported?

Reply:
Have added more info to the doc in d95a18f654